presto-orc/src/main/java/com/facebook/presto/orc/OrcSelectiveRecordReader.java

```diff
@@ -876,6 +859,24 @@ private void initializeTmpErrors(int positionCount)
         }
     }
 
+    private void initializePositions(int batchSize)
+    {
+        if (positions == null || positions.length < batchSize) {
+            positions = new int[batchSize];
```

Comment:
You can just write as 

int[] positions = IntStream.range(0, batchSize).toArray();

Reply:
Sorry, but I don't see a good reason to wrap such a simple thing into syntax sugar like streams. Current implementation performs well.